interface/system/config_backup.rst

```diff
@@ -51,7 +56,7 @@ states to the list of saved states in the future.
 This feature is found in the **Config Backups** section on the **System** page.
 
 ..  image:: /images/interface/system/config_backup/conf_backup.png
-    :width: 100%
+    :width: 90%
```

Comment:
@Hooverdan96 I don't see why this width was change. But no worries as not a show stopper.

Reply:
I felt it looked better on my rendering of the screen :)